code_challenges/Account_Generator.py

```diff
@@ -0,0 +1,60 @@
+import random
+
+
+def account_generator():
+    student_names = []
+    student_id_numbers = []
+    student_email_addresses = []
+    email_name = []
+
+    for i in range(int(input("enter how many students you have: "))):
+        student_names.append(input("Enter student name: "))
+
+    for name in student_names:
+        student_id_number = random.randint(111111, 999999)
+        student_id_numbers.append(student_id_number)
+        while student_id_number in student_id_numbers:
+            student_id_number = random.randint(111111, 999999)
+        student_name = name.split(" ")
```

Comment:
```
for name in student_names:
    student_id_number = random.randint(111111, 999999)
    student_id_numbers.append(student_id_number)  # <--- When would you want this to happen?
    while student_id_number in student_id_numbers:
        student_id_number = random.randint(111111, 999999)
    student_name = name.split(" ")
```
I think after you are sure it isn’t already in the list.  So probably something like:
```
ffor name in student_names:
    student_id_number = random.randint(111111, 999999)
    while student_id_number in student_id_numbers:
        student_id_number = random.randint(111111, 999999)
    student_id_numbers.append(student_id_number) 
    student_name = name.split(" ")
```